xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-pageobjects/src/main/java/org/xwiki/administration/test/po/TemplatesAdministrationSectionPage.java

```diff
@@ -74,19 +74,21 @@ public TemplateProviderInlinePage createTemplateProvider(String space, String pa
         documentPicker.toggleLocationAdvancedEdit();
         documentPicker.setParent(space);
         documentPicker.setName(page);
-        
-        // Livevalidation blocks the form submission until the validated fields are valid. Until the form is validated,
-        // Livevalidation prevents the submission button to be clicked.
-        // On some rare cases, the click of the button is done too early and the form submission is prevented.
-        // To be sure that the form is validated before clicking, wait for the two fields validation messages to be 
-        // displayed before clicking.
-        getDriver().waitUntilCondition(input ->
-            getDriver().findElementsWithoutWaiting(By.cssSelector("form .LV_validation_message.LV_valid")).size() == 2);
-        this.createButton.click();
```

Comment:
I'm not sure it is a good idea to remove the wait for the live validation. This is a different issue around blocking the submission of the button, not that something doesn't wait afterwards. Iirc the screenshots clearly showed that the submission didn't happen (you could check if there is a recent run with the failure).